package.json

```
@@ -41,11 +54,12 @@
 		"watch:html": "npx @11ty/eleventy --config=_build/eleventy.js --serve",
 		"build:js": "rollup -c _build/rollup.config.js",
 		"watch:js": "rollup -c _build/rollup.config.js --watch",
+		"build:ts": "tsc && node ./scripts/copy-declarations.js",
```

When I run `build:ts` I get compilation errors so the copy-declarations script is never run.

```suggestion
		"build:ts": "node ./scripts/copy-declarations.js && tsc",
```